binders/kafka-binder/pom.xml

```diff
@@ -20,10 +20,6 @@
 	</scm>
 	<properties>
 		<java.version>1.8</java.version>
-		<spring-kafka.version>2.8.5</spring-kafka.version>
```

Comment:
2 of these were unused prior to this MR (`spring-integration-kafka.version` and `spring-cloud-stream.version`). 
The other 2 are no longer needed as we defer to SB BOM for dependency mgmt.